examples/cpp/io/save_map_to_stream.cc

```diff
@@ -0,0 +1,17 @@
+#include <strstream>
+
+#include <wavemap/io/stream_conversions.h>
+
+int main(int, char**) {
+  // Create an empty map for illustration purposes
+  wavemap::HashedWaveletOctreeConfig config;
+  wavemap::HashedWaveletOctree map(config);
+
+  // Create an output stream for illustration purposes
+  std::ostrstream output_stream;
```

Comment:
Could you also use a `ofstream` as output stream so it's a bit easier for people to understand? Not sure how many people ever used `strstream` before :-)